cvat/settings/base.py

```diff
@@ -515,3 +515,6 @@ def add_ssh_keys():
 TUS_MAX_FILE_SIZE = 26843545600 # 25gb
 TUS_DEFAULT_CHUNK_SIZE = 104857600  # 100 mb
 
+# used to make request secure if X-Forwarded-Proto: 'https' header is specified by our proxy
+# https://docs.djangoproject.com/en/2.2/ref/settings/#secure-proxy-ssl-header
```

Comment:
I think the following link will also be helpful https://github.com/traefik/traefik/blob/master/docs/content/getting-started/faq.md#what-are-the-forwarded-headers-when-proxying-http-requests (or https://doc.traefik.io/traefik/getting-started/faq/#what-are-the-forwarded-headers-when-proxying-http-requests)

Reply:
Done